### ISSUE TITLE
ERROR: process is translated under Rosetta. Attempting to replace emulator installation.

### ISSUE THREAD

**Issue by thalwani** [renative-org/renative#780]
**Describe the bug**
Hi, 

I can't run on android emulator on my MAC MINI M1 arm 64. I can run react native app, but rnv is not working... The ANDROID_SDK path is wrong, it is not reading from my .zshrc file...

**To Reproduce**

1. Run command 'rnv run -p android'
2. See error
/Users/XXX/Android/emulator/emulator -list-avds 

FAILED with ERROR: 

emulator: ERROR: process is translated under Rosetta. Attempting to replace emulator installation.
emulator: Replacing via command: /Users/XXX/Android/emulator/darwin-aarch64-replace.sh (downloading ~120 MB)...
sh: /Users/XXX/Android/emulator/darwin-aarch64-replace.sh: No such file or directory
emulator: Replacement done. Please relaunch the emulator. You will also need to be using an Apple Silicon-compatible system image. Check the release updates blog (https://androidstudio.googleblog.com/) for more details.

Command failed with exit code 255 (Unknown system error -255): /Users/XXX/Android/emulator/emulator -list-avds

**Expected behavior**
Run app on android emulator.

**Desktop (please complete the following information):**

-   OS: MacOs Big Sur ARM M1 chip
-   Node Version v16.13.2
-   RNV Version 0.34.0

Thank you for your help.

**Comment by mihaiblaga89** [comment]
@thalwani  While RNV should detect that and try to either find the existing SDK based on the common env variables or ask you if it can install it, you can change the config yourself. It's located in `~/.rnv/renative.json`. Change the path of the SDK there and you should be good to go. Can you post the full output of the command first so I can try to debug why the "helping" part is not happening?

**Comment by thalwani** [comment]
Thank you for your response. 

I had to add this in the renative.json file : 
```
    "sdks": {
        "ANDROID_SDK": "/Users/toufichalwani/Library/Android/sdk", // (<USER> doesn't work)
        "ANDROID_NDK": "/Users/<USER>/Library/Android/sdk/ndk-bundle",
        "IOS_SDK": "No need. Just install Xcode",
        "TIZEN_SDK": "/Users/<USER>/tizen-studio",
        "WEBOS_SDK": "/Users/<USER>/Library/webOS_TV_SDK",
        "KAIOS_SDK": "/Applications/Kaiosrt.app"
    }
```

but it was searching the emulator in the folder /Users/toufichalwani/Library/Android/sdk/emulator/emulator while it sit in the folder /Users/toufichalwani/Library/Android/sdk/tools/emulator, so I copied this file. Now it is working, but when the application launch it automatically crashes. (See attachment). On iOs, I can't see the app in simulator.

I can't send you the whole error sorry, I can't reproduce it since I change the sdk path. It seems Renative has kept the new working sdk path in "memory".

Thanks.


![Capture d’écran 2022-01-15 à 06 47 29](https://user-images.githubusercontent.com/1224656/149611129-7a93c53a-7fbb-4b3b-b0fc-4d99b6c5f4a9.png)
